lib/server.js

```diff
@@ -86,8 +86,16 @@ FastBootServer.prototype.insertIntoIndexHTML = function(title, body, head) {
 };
 
 FastBootServer.prototype.handleSuccess = function(res, path, result, startTime) {
-  this.log(200, 'OK ' + path, startTime);
-  res.send(this.insertIntoIndexHTML(result.title, result.body, result.head));
+  var contentFor = result.contentFor;
+  var headers = result.headers;
+
+  this.log(result.statusCode, 'OK ' + path, startTime);
```

Comment:
Yeah, like I said, I avoided doing status code parsing in this PR. We can do it in a separate PR. We can bring it up in the fastboot sync tomorrow or next week.